interbtc-spec/docs/source/spec/security.rst

```diff
@@ -61,6 +55,8 @@ Indicates ths status of the BTC Parachain.
 
 * ``SHUTDOWN: 2`` - BTC Parachain operation fully suspended. This can only be achieved via manual intervention by the Governance Mechanism.
 
+.. _errorCode:
+
 ErrorCode
```

Comment:
Note that there is now only one possible `ErrorCode` since the removal of `BTC_RELAY_OFFLINE`.